cogs/moderation.py

```
@@ -0,0 +1,258 @@
+        description="Give a warning to a member",
+        usage="warn <member> [reason]"
+    )
+    @commands.has_permissions(manage_messages=True)
+    async def warn(self, ctx, member: discord.Member, *, reason: str = "No reason provided"):
+        """Warn a member"""
+        await self.bot.db.add_warning(ctx.guild.id, member.id, ctx.author.id, reason)
+        
+        try:
+            await member.send(f"You have been warned in {ctx.guild.name}. Reason: {reason}")
+        except:
+            pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)